src/main/scala/uk/gov/hmrc/govukfrontend/views/implicits/RichStringSupport.scala

```diff
@@ -16,10 +16,47 @@
 
 package uk.gov.hmrc.govukfrontend.views.implicits
 
-trait RichStringSupport {
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Text}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.fieldset.{Fieldset, Legend}
+import uk.gov.hmrc.govukfrontend.views.viewmodels.hint.Hint
+import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.Key
+import uk.gov.hmrc.govukfrontend.views.viewmodels.table.HeadCell
+import uk.gov.hmrc.govukfrontend.views.viewmodels.tag.Tag
 
+trait RichStringSupport {
   implicit class RichString(s: String) {
 
+    def toHint(args: Any*)(implicit messages: Messages): Hint = Hint(content = Text(messages(s, args)))
```

Comment:
This demonstrates adding an extension method to convert rather than relying on implicit conversions - less magical. Also allows for addition of message arguments.